site/styles/_shopping.scss

```diff
@@ -132,7 +132,8 @@
   }
 }
 
-.s72-shopping-info-meta {
+.s72-shopping-info-meta,
+.s72-shopping-info-live-event {
```

Comment:
Can we please remove this class and instead just use `s72-shopping-info-meta` [on this relish line](https://github.com/indiereign/shift72-web/pull/350/files#diff-c7a9caf8a35cd8436d3c47ab8a3d6d3af3bacb9ffa715747d604da982c30a1a1R98) instead?

I would personally prefer to discourage this where possible as it convolutes the code as opposed to making it cleaner and easier to read/debug. It would also be more performant.

Reply:
I gave it a different class because that's the pattern with the other shopping modal header variants.  I suspect they have different classes so they can be targeted separately for client customisations... if they all have the same class then we lose that functionality.

Reply:
Can we go with this instead? `class="s72-shopping-info-meta live-event"`

Reply:
Yeah I prefer that!